<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_MemberManage">
         <property name="text">
          <string>명단 관리</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_TrainingManage">
         <property name="text">
          <string>체력단련일지</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_RecordManage">
         <property name="text">
          <string>체력측정기록</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_SquadManage">
         <property name="text">
          <string>편성표 작성</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Inbody">
         <property name="text">
          <string>인바디 기록</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_PictureManage">
         <property name="text">
          <string>증명사진 관리</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_setting">
         <property name="text">
          <string>설정</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="Page_MemberManage">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="btn_NewMember">
                <property name="text">
                 <string>새로 만들기</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_EditMember">
                <property name="text">
                 <string>편집</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QListWidget" name="list_Memberlist"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Page_Training">
          <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0" columnstretch="0">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="2" column="0">
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>체단 일자</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radio_3daystraining">
                  <property name="text">
                   <string>주 3일 (월화수)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radio_5daystraining">
                  <property name="text">
                   <string>주 5일 </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>출석 확인란</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radio_include_attendance">
                  <property name="text">
                   <string>포함</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radio_exclude_attendance">
                  <property name="text">
                   <string>비포함</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>타공라인</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="check_MakeHole">
                  <property name="text">
                   <string>포함 / 비포함</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>기준일</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="DE_startdate">
                  <property name="calendarPopup">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="btn_TrainingPrint">
                <property name="text">
                 <string>출력</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>미리보기</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTableView" name="Preview_Training"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Page_Record">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="btn_NewRecord">
               <property name="text">
                <string>새로 만들기</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_EditRecord">
               <property name="text">
                <string>편집</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QListWidget" name="list_Record"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>도움말</string>
    </property>
    <addaction name="actionView_AppInfo"/>
    <addaction name="actionhelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionView_AppInfo">
   <property name="text">
    <string>프로그램 정보</string>
   </property>
   <property name="shortcut">
    <string>R, Backspace</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>사용 설명서</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
